tree/ntuple/v7/inc/ROOT/RNTupleSerialize.hxx

```diff
@@ -72,6 +73,8 @@ public:
 
    static constexpr DescriptorId_t kZeroFieldId = std::uint64_t(-2);
 
+   static constexpr int64_t kSuppressedColumnMarker = std::numeric_limits<std::int64_t>::min();
```

Comment:
This is not `uint64_t(-1)` as described in the specification!

Reply:
Thanks! Good catch, updated the specs.